MODULE.bazel

```diff
@@ -233,7 +224,7 @@ maven.install(
     ]]
 ]
 
-use_repo(maven, "maven", "unpinned_maven")
```

Comment:
https://github.com/bazel-contrib/rules_jvm_external/blob/d45e59ee582c2999dabbccab43320ce92d5bb0ee/private/rules/coursier.bzl#L1316

`unpinned_maven` is used to update the maven lockfile, are we sure we want to remove it?

Reply:
yeah, now you can just run `REPIN=1 bazel run @maven//:pin` (the "pinned" maven now generates an alias into "unpinned" maven)

Reply:
Nice!